### ISSUE TITLE
Migrate to pyproject.toml

### ISSUE THREAD

**Issue by mikegerber** [OCR-D/ocrd_calamari#103]
- [x] Migrate to pyproject.toml
  - [x] setup.py
  - [x] .coveragerc
  - [x] .ruff.toml
- [x] #85
- [x] Review keywords
- [x] Review classifiers
- [x] license.file?
- [x] Review mypy (copied from dinglehopper's config - do we want this here?)
- [x] Check if the coverage config is working
- [x] Remove workaround for core#1262

I'm going to use setuptools-ocrd for this (as the second project using it). If OCR-D decides against using it I can still change the project back to whatever OCR-D agrees upon.

**Comment by mikegerber** [comment]
Ah, now I remember why I moved to a "src-layout" with dinglehopper:

```
      error: Multiple top-level packages discovered in a flat-layout: ['repo', 'ocrd_calamari', 'euler_rechenkunst01_1738_selected_pages_ocr'].
```

**Comment by mikegerber** [comment]
> Ah, now I remember why I moved to a "src-layout" with dinglehopper:
> 
> ```
>       error: Multiple top-level packages discovered in a flat-layout: ['repo', 'ocrd_calamari', 'euler_rechenkunst01_1738_selected_pages_ocr'].
> ```

Now using the alternative solution:

```
[tool.setuptools.packages.find]
where = ["."]
include = ["ocrd_calamari"]
```

**Comment by mikegerber** [comment]
PyPI's Trove classifier selection seems to be a bit lacking... Leaving as is with the most fitting ones.

**Comment by mikegerber** [comment]
No more obvious problems; PR #115 can be merged after the workaround for core#1262 is removed.
